### PR TITLE
Implement Lab integration of Fracking model

### PR DESCRIPTION
**Pull request by pjanik**
Yet another HAS-Lab integration. Similar to previous ones, the main difference is that model view is scrolling and I had to modify a bit existing model and controls.

### CODE REVIEW

**Comment by scytacki** [comment]
@psndcsrv Can you take a look at this one?

File: fracking/js/fracking-controls.coffee [hunk 1]
```diff
@@ -188,24 +189,26 @@ class FrackingControls
...
+      # start the graph with four lines, each at 0,0
+      @outputGraph.addSamples [0, 0, 0, 0]
 
-    $(document).on FrackingModel.YEAR_ELAPSED, =>
-      killed = [0,0,0,0]
-      killed[3] = ABM.model.killed
+      $(document).on FrackingModel.YEAR_ELAPSED, =>
+        killed = [0,0,0,0]
+        killed[3] = ABM.model.killed
+        console.log(killed[3])
```
Comment (psndcsrv) [inline fracking/js/fracking-controls.coffee]: Probably don't need to log every time something is killed...

File: fracking/js/lab-integration.coffee [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+        model.leaks = content.value
+      when 'drillDirection'
+        model.drillDirection = content.value
+
+
+  getOutputs = ->
+    result =
+      year: model.getYear(),
+      removeFluidPossible: false,
+      fillPossible: false,
```
Comment (psndcsrv) [inline fracking/js/lab-integration.coffee]: The commas are unnecessary here, but that's a style choice (I tend to leave them out in these situations)...

**Comment by psndcsrv** [comment]
Looks fine to me. Just a couple lint-oriented things.

**Comment by pjanik** [comment]
Thanks! I've fixed listed issues, so I'm merging and closing this PR.

**Pull request merged**